modules/administration_manual/pages/installation/system_requirements.adoc

```diff
@@ -28,7 +28,7 @@ we officially recommend and support:
 |
 * MySQL or MariaDB 5.5+
 * Oracle 11g
-* PostgreSQL
+* PostgreSQL 9.0+
```

Comment:
Is it clear what this means?
PostgreSQL10 is not (yet) supported
But to me the ``+`` sign means 9.0 or later, which includes PostgreSQL10, PostgreSQL11...

Reply:
Well said. I'll correct.

Reply:
Just adjusted the text to be clear that only version 9's supported.